internal/meta/branch.go

```diff
@@ -66,6 +67,39 @@ func ReadBranch(repo *git.Repo, branchName string) (Branch, bool) {
 	return branch, true
 }
 
+func ReadAllBranches(repo *git.Repo) (map[string]Branch, error) {
```

Comment:
some comments / documentation here would be nice explaining that this method is reading through all meta information stored locally to identify all stacked branches.